terminology.md

```diff
@@ -261,17 +261,23 @@ A realm consists of a set of [intrinsic](#Intrinsic) objects, an ECMAScript glob
 
 ### Intrinsic
 #### Definition
-Built-in objects that are explicitly referenced by the algorithms of the ECMA262 specification.
+Built-in objects (usually realm-specific) that are explicitly referenced by the algorithms of the ECMA262 specification.
 #### Example
-`[]`
+%ForInIteratorPrototype%
 #### Reference
 [ECMA262 Spec](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects)
 
 ### Primordial
 #### Definition
-Global, mutable objects that must exist before any code runs.
+The subset of [intrinsic](#Intrinsic) objects that are accessible to ECMAScript code.
 #### Example
-`%ArrayPrototype%`
+%Array% is accessible as `Array`.
+
+### Undeniable
+#### Definition
+The subset of [primordial](#Primordial) objects that are accessible to ECMAScript code without reference to named bindings (i.e., by means of syntax alone).
+#### Example
+%ArrayPrototype% is accessible as `[].__proto__`.
```

Comment:
yes.

Reply:
What I find missing from these definitions is "Well-known intrinsic" and an explanation of the %foo% notation. 

Prior to ES6, the specification didn't have a concise way  to refer to things such as  "the original value of the `Array` property of the global object" or "the original value of the `toString` property of the object that is the value of the `prototype` property of the object that is the original value of the `Object` property of the  global object".

The %names% in the well-known intrinsic object table provide a concise way for specification text to unambiguously reference such objects; eg,`%Array%` or `%ObjProto_toString%`.  Only intrinsics that actually need to be mentioned in specification text are included in that table.  Well-knownness  has nothing to do with whether or not such an object is also "undeniable".  A well-known intrinsic object is always available to the specification algorithms, regardless anything ECMAScript code might do.

Reply:
The page says nothing about "well-known" values either before or after changes proposed here. This PR is about providing clarity to some terms that have been used for naming significant classes of specification-defined ECMAScript values (but so imprecisely that it has disrupted discussions).
Point taken about the %-notation, though... I'll try to address that.